src/sentry/notifications/utils/__init__.py

```diff
@@ -533,3 +540,64 @@ def parameters(self) -> List[str]:
         return [
             "{{{}: {}}}".format(key, ",".join(values)) for key, values in all_parameters.items()
         ]
+
+
+class ConsecutiveDBQueriesProblemContext(PerformanceProblemContext):
+    def __init__(
+        self,
+        problem: PerformanceProblem,
+        spans: Union[List[Dict[str, Union[str, float]]], None],
+        event: Event | None,
+    ):
+        PerformanceProblemContext.__init__(self, problem, spans)
+        self.event = event
+
+    def to_dict(self) -> Dict[str, Any]:
+        return {
+            "span_evidence_key_value": [
+                {"key": "Transaction", "value": self.transaction},
+                {"key": "Starting Span", "value": self.starting_span},
+                {
+                    "key": "Parallelizable Spans",
+                    "value": self.parallelizable_spans,
+                    "is_multi_value": True,
+                },
+            ],
```

Comment:
Having generic way to render items in the list makes a lot of sense, but `"span_evidence_key_value"` is pretty awkward. Maybe the `to_dict` approach doesn't make sense anymore. The method could be a property called `values` for example, or you can make the whole data class an iterable, so you can do `for item in context` in the Django template

Reply:
Also maybe a bit weird to add this _on top_ of the existing template, it adds a bunch of code that the current template doesn't benefit from, but NBD as long as it's consolidated soon

Reply:
Yeah, I think thats to consider in future cleanup tasks.